src/Raven.Client/Constants.cs

```diff
@@ -69,8 +69,6 @@ private Windows()
                 }
 
                 public static readonly int MaxPath = (int)typeof(Path).GetField("MaxLongPath", BindingFlags.Static | BindingFlags.NonPublic).GetValue(null);
-
-                public static readonly string[] ReservedFileNames = { "con", "prn", "aux", "nul", "com1", "com2", "com3", "com4", "com5", "com6", "com7", "com8", "com9", "lpt1", "lpt2", "lpt3", "lpt4", "lpt5", "lpt6", "lpt7", "lpt8", "lpt9", "clock$" };
```

Comment:
Why did you remove this?

Reply:
I wasn't used here. There is another place we have those constants.